tests/app/item/DVDTest.java

```diff
@@ -0,0 +1,40 @@
+package app.item;
+
+import app.catalogue.Catalogue;
+import app.catalogue.CataloguePopulator;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * This class will test the functionality of the toString method for the DVD object.
+ *
+ * @author chad_olsen
+ * @since 2017/06/28.
+ */
+public class DVDTest {
+    @Test
+    public void toStringTest() throws Exception {
+        Item dvd = new DVD("Titanic", "Drama", "2hrs", "Leonardo di Caprio", "Kate Winslet");
+
+        Catalogue catalogue = CataloguePopulator.populateCatalogue();
+        String catalogueString = catalogue.getCatalogueMap().get(Type.DVD).get(0).toString();
+        assertEquals(catalogueString, dvd.toString());
+
+        DVD newDVD = new DVD("Ice Age", "Animation", "1hr30mins", "Manny", "Sid");
+        catalogue.addItemToList(newDVD);
+        catalogueString = catalogue.getCatalogueMap().get(Type.DVD).get(2).toString();
+        assertEquals(catalogueString, newDVD.toString());
+    }
+
+    @Test
+    public void toString_nullTest() {
+        DVD dvd = new DVD();
+
+        Catalogue catalogue = new Catalogue();
+        catalogue.addItemToList(dvd);
+        String catalogueString = catalogue.getCatalogueMap().get(Type.DVD).get(0).toString();
+        assertEquals(catalogueString, dvd.toString());
```

Comment:
You are comparing the toString() values from the exact same object, they were just retrieved differently. Rather build up the string manually and then compare it to the value returned on the DVD object so that we can ensure the value hasn't changed or this test would never ever fail.